src/Mutation/PostObjectCreate.php

```diff
@@ -295,6 +295,17 @@ public static function mutate_and_get_payload( $post_type_object, $mutation_name
 				throw new UserError( __( 'The object failed to create', 'wp-graphql' ) );
 			}
 
+			/**
+			 * Fires after a single term is created or updated via a GraphQL mutation
+			 *
+			 * The dynamic portion of the hook name, `$taxonomy->name` refers to the taxonomy of the term being mutated
+			 *
+			 * @param int    $post_id       Inserted post ID
+			 * @param array  $args          The args used to insert the term
+			 * @param string $mutation_name The name of the mutation being performed
+			 */
+			do_action( "graphql_insert_{$post_type_object->name}", $post_id, $input, $mutation_name );
```

Comment:
@markkelnar I think it might be better to centralize an action like this for all mutations to use. 

I'm thinking we could maybe put this action here: https://github.com/wp-graphql/wp-graphql/blob/develop/src/Registry/TypeRegistry.php#L1160

That way all mutations are hookable, including mutations created by 3rd party plugins, etc.

I'm thinking it might be good to have a _before_ and _after_ action, like: 

```php

$user_input = $args['input'];

$filtered_input = apply_filters( 'graphql_mutation_input', $user_input, $context, $info, $whatever_else_seems_relevant )

$response = call_user_func( $mutateAndGetPayload, $filtered_input, $context, $info );

$filtered_response = apply_filters( 'graphql_mutation_resonse', $response, $filtered_input, $user_input, $context, $info );

// then return the filtered response. 
```

this would allow plugins to hook in and modify filters _before_ and _after_ the execution of _any_ mutation's resolver.

Reply:
Interesting.  My change was to mimic an [action that already exists](https://github.com/wp-graphql/wp-graphql/blob/develop/src/Mutation/PostObjectUpdate.php#L175) in the post object update.

Reply:
Ya, that was one of the first mutations in the codebase, and the filter was added for some specific needs. Now that some patterns have emerged, it makes more sense to centralize filters like this that can be used across the board. 

I'm not 💯 opposed to adding hooks/filters for specific mutations when it makes sense, but ideally I'd like to centralize things that can be centralized, and I _think_ this is one of those cases?